webapp/channels/src/actions/views/textbox.js

```diff
@@ -23,3 +23,17 @@ export function setShowPreviewOnEditChannelHeaderModal(showPreview) {
         showPreview,
     };
 }
+
+export function setShowPreviewOnChannelSettingsHeaderModal(showPreview) {
```

Comment:
the current implementation we had for the edit channel header textbox worked this way, attaching to the state the hide/show preview. I think with the new advanced component it could be handled with the react context. I can work on that in a new PR.